pkg/view/component/row_detail.go

```diff
@@ -29,46 +34,56 @@ type ExpandableRowDetail struct {
 
 // ExpandableDetailConfig is a configuration for an expandable row detail.
 type ExpandableDetailConfig struct {
-	Replace bool      `json:"replace"`
-	Body    Component `json:"body"`
+	Replace bool        `json:"replace"`
+	Body    []Component `json:"body"`
 }
 
-// Unmarshal unmarshals an expandable row detail config from JSON.
+// UnmarshalJSON unmarshals an expandable row detail config from JSON.
 func (e *ExpandableDetailConfig) UnmarshalJSON(data []byte) error {
 	x := struct {
-		Body    *TypedObject `json:"body"`
-		Replace bool         `json:"replace"`
+		Body    []TypedObject `json:"body"`
+		Replace bool          `json:"replace"`
```

Comment:
Just a question, what is the purpose of  the `Replace` attribute? Couldn't figure it out

Reply:
As the row is expanded it can show the row information together with the row details or it could hide the row info and show only the row details
![image](https://user-images.githubusercontent.com/6991432/114577829-fc062300-9c41-11eb-8631-efabd6f582d3.png)